Service/ContentTypeService.php

```diff
@@ -426,10 +430,30 @@ public function getAllTypes()
         return implode(',', array_keys($this->contentTypeArrayByName));
     }
 
-    public function contentTypeFromJson(string $json, Environment $environment): ContentType
+
+    public function updateFromJson(ContentType $contentType, string $json, bool $isDeleteExitingTemplates, bool $isDeleteExitingViews): void
+    {
+        $this->deleteFields($contentType);
```

Comment:
What happens if your program would be interrupted at this stage? Your content types fields would be removed, and the update never executed..